zerver/lib/actions.py

```diff
@@ -768,6 +822,7 @@ def recipient_for_emails(emails, not_forged_mirror_message,
         return get_recipient(Recipient.PERSONAL, list(recipient_profile_ids)[0])
 
 def already_sent_mirrored_message_id(message):
+    # type: (Message) -> int
```

Comment:
I think passing None for a value that isn't explicitly tagged as `Optional` isn't enforced in mypy yet, but will be soon, so we'll definitely need these Optionals in the annotations eventually.